dao/postgres/pg-utils.js

```diff
@@ -413,8 +422,14 @@ class PgUtils {
         debug(`addFilterOptions()`);
         for (let fieldName in filter) {
             fields.push(fieldName);
-            values.push(`%${filter[fieldName]}%`);
-            operators.push('ILIKE');
+            if (Array.isArray(filter[fieldName])) {
+                let vals = filter[fieldName].join('|');
+                values.push(`(${vals})`);
+                operators.push('SIMILAR TO');
+            } else {
+                values.push(`%${filter[fieldName]}%`);
+                operators.push('ILIKE');
```

Comment:
What about the issue where a reviewer has group "review" and there are other groups like "superreview"? This is still not addressed with this, is it?